src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Base/App.default.cs

```diff
@@ -48,39 +52,89 @@ protected async override void OnLaunched(LaunchActivatedEventArgs args)
 							LogLevel.Information :
 							LogLevel.Warning);
 				}, enableUnoLogging: true)
-#if useserilog
+#if use-serilog
 				.UseSerilog(consoleLoggingEnabled: true, fileLoggingEnabled: true)
 #endif
 #endif
-#if configuration
+#if use-configuration
 				.UseConfiguration(configure: configBuilder =>
 					configBuilder
-						.EmbeddedSource<App>()
+						.EmbeddedSource<MainPage>()
```

Comment:
Change MainPage back to App

Reply:
It was changed to MainPage on purpose because it needs to actually work